### ISSUE TITLE
Your requirements could not be resolved to an installable set of packages.

### ISSUE THREAD

**Issue by gigi** [spiechu/lazy-pimple#2]
Hi!

Thanx for this lib! Great job!
Seems that package cant work with proxy-manager ~2.0.0

```
vagrant@vagrant-ubuntu-trusty-64:/srv/www/silex.local$ composer require spiechu/lazy-pimple
Using version ^0.4.0 for spiechu/lazy-pimple
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for spiechu/lazy-pimple ^0.4.0 -> satisfiable by spiechu/lazy-pimple[0.4].
    - Conclusion: don't install ocramius/proxy-manager 2.0.4
    - spiechu/lazy-pimple 0.4 requires ocramius/proxy-manager ~1.0 -> satisfiable by ocramius/proxy-manager[1.0.0, 1.0.1, 1.0.2].
    - Can only install one of: ocramius/proxy-manager[1.0.0, 2.0.4].
    - Can only install one of: ocramius/proxy-manager[1.0.1, 2.0.4].
    - Can only install one of: ocramius/proxy-manager[1.0.2, 2.0.4].
    - Installation request for ocramius/proxy-manager (locked at 2.0.4) -> satisfiable by ocramius/proxy-manager[2.0.4].


Installation failed, reverting ./composer.json to its original content.
```

Do you have any plans for update?

**Comment by spiechu** [comment]
Hi, thanks for finding this. I need to take a look at breaking changes between proxy-manager 1 and 2.
From what I can see proxy manager 2.0 requires PHP 7.0, but proxy manager 2.1 requires PHP 7.1.

**Comment by gigi** [comment]
Yes. I use 2.0.4 with php 7.0. Works fine

**Comment by spiechu** [comment]
I've just released 0.5 expanding proxy manager requirements. Please update your project.

Closing issue, will reopen if needed.
